agent/pkg/tunnel/module.go

```diff
@@ -46,21 +49,45 @@ func newTunnelAgent(c *config.TunnelAgentConfig, ifm *informers.Manager, mode Tu
 
 	controller.Init(ifm)
 
-	privateKey, err := acl.GetPrivateKey(c.TunnelACLConfig)
-	if err != nil {
-		return nil, fmt.Errorf("Failed to get private key")
+	var aclManager acl.Manager
+	if c.EnableSecurity {
+		// fetch the cloudcore token
+		content, err := ioutil.ReadFile(meshConstants.CaServerTokenPath)
+		if err != nil {
+			klog.Fatalf("caServerToken file %s not exist", meshConstants.CaServerTokenPath)
+		} else {
+			klog.Infof("fetch token success, %s", string(content))
```

Comment:
token is secret, should not be logging explictly

Reply:
fix